src/sentry/api/utils.py

```diff
@@ -498,10 +498,12 @@ def id_or_slug_path_params_enabled(
         return False
 
     # EA option for endpoints where organization is not available
-    if convert_args_class in options.get("api.id-or-slug-enabled-ea-endpoints"):
-        return True
+    if convert_args_class:
+        if convert_args_class in options.get("api.id-or-slug-enabled-ea-endpoints"):
+            return True
+        return False
```

Comment:
```suggestion
        return convert_args_class in options.get("api.id-or-slug-enabled-ea-endpoints"):
```

Could golf this down.